src/daft-sql/src/functions.rs

```diff
@@ -242,14 +243,21 @@ impl<'a> SQLPlanner<'a> {
         // assert using only supported features
         check_features(func)?;
 
+        fn get(
+            fns: &Lazy<SQLFunctions>,
```

Comment:
Is it necessary to pass in `SQL_FUNCTIONS` into here? I believe you can access it directly in the function.

Reply:
Whoops, you're right! Will fix in upcoming commit.

Reply:
Updated.